code/_onclick/hud/robot.dm

```diff
@@ -1,241 +1,241 @@
-/obj/screen/robot
```

Comment:
Alright I'll get on that now, I did it with raw paste and that didn't work so I'll try this method too